src/analysis/retail/warlock/demonology/modules/tier/TWW3DiabolistTierSet.tsx

```diff
@@ -0,0 +1,133 @@
+ * Consuming Demonic Art flings your Demonic Oculi at your current target,
+ * exploding for Fire damage to the target and nearby enemies.
+ *
+ * 4pc: Demonic Oculi analyze the battle while active and deliver information
+ * to you as they explode, increasing your Intellect by 2% for 10 sec.
+ */
+class TWW3DiabolistTierSet extends Analyzer {
+  has2Piece: boolean;
+
+  // Tracking for 2pc
+  oculusSummons = 0;
+  oculusExplosions = 0;
+  oculusDamage = 0;
+  handOfGuldanCasts = 0;
+  fullPowerHandOfGuldanCasts = 0;
+
+  constructor(options: Options) {
+    super(options);
+    // Check if player has TWW3 tier set
+    this.has2Piece = this.selectedCombatant.has2PieceByTier(TIERS.TWW3);
+
+    if (!this.has2Piece) {
+      this.active = false;
+      return;
+    }
+
+    // 2pc events
+    this.addEventListener(
+      Events.cast.by(SELECTED_PLAYER).spell(SPELLS.HAND_OF_GULDAN_CAST),
+      this.onHandOfGuldan,
+    );
+    // Track Oculus buff applications (each buff = 1 full power HoG)
+    this.addEventListener(
+      Events.applybuff.to(SELECTED_PLAYER).spell(SPELLS.DEMONIC_OCULUS_BUFF),
+      this.onOculusBuff,
+    );
+    // Track when Oculus buffs are removed (when they explode)
+    this.addEventListener(
+      Events.removebuff.to(SELECTED_PLAYER).spell(SPELLS.DEMONIC_OCULUS_BUFF),
+      this.onOculusExplosion,
+    );
+    this.addEventListener(
+      Events.damage.by(SELECTED_PLAYER).spell(SPELLS.EYE_BLAST),
+      this.onOculusDamage,
+    );
+  }
+
+  onHandOfGuldan(event: CastEvent) {
+    // Track all Hand of Gul'dan casts
+    this.handOfGuldanCasts += 1;
+    // Full power casts are tracked via the Oculus buff application
+  }
+
+  onOculusBuff(event: ApplyBuffEvent) {
+    // Each buff application means a full power (3-shard) Hand of Gul'dan was cast
+    this.fullPowerHandOfGuldanCasts += 1;
+    this.oculusSummons += 1;
+  }
+
+  onOculusExplosion(event: RemoveBuffEvent) {
+    // When an Oculus buff is removed, it explodes
+    this.oculusExplosions += 1;
+  }
+
+  onOculusDamage(event: DamageEvent) {
+    // Activate this module when we see Eye Blast or Demonic Oculus damage (Diabolist hero talent)
+    this.active = true;
+
+    this.oculusDamage += event.amount + (event.absorbed || 0);
+  }
+
+  statistic() {
+    // Only show this statistic if we detected Diabolist hero talent
+    if (!this.active || this.oculusDamage === 0) {
+      return null;
+    }
+
+    return (
+      <Statistic
+        position={STATISTIC_ORDER.OPTIONAL(1)}
+        size="flexible"
+        category={STATISTIC_CATEGORY.ITEMS}
+        tooltip={
+          <>
+            <strong>2-piece:</strong>
+            <br />
+            Hand of Gul'dan Casts: {this.handOfGuldanCasts}
+            <br />
+            Full Power (3-shard) Casts: {this.fullPowerHandOfGuldanCasts}
+            <br />
+            Oculus Summons: {this.oculusSummons}
+            <br />
+            Total Explosions: {this.oculusExplosions}
+          </>
```

Comment:
use a `<ul>` for this instead of `<br/>`s, e.g.

```suggestion
          <>
            <strong>2-piece:</strong>
            <ul>
              <li>Hand of Gul'dan Casts: {this.handOfGuldanCasts}</li>
              <li>Full Power (3-shard) Casts: {this.fullPowerHandOfGuldanCasts}</li>
              <li>Oculus Summons: {this.oculusSummons}</li>
              <li>Total Explosions: {this.oculusExplosions}</li>
            </ul>
          </>
```